### PR TITLE
Added update for model package

### PR DESCRIPTION
**Pull request by keshav-chandak**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: c8980a90026e78a3f5b47d186302d0f8dcc6f68d
* Result: FAILED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=b2d5f346-056c-487f-b449-80756a5bdc40%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: c8980a90026e78a3f5b47d186302d0f8dcc6f68d
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ad930442-0682-453e-a5ed-1503e1384dd4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: c8980a90026e78a3f5b47d186302d0f8dcc6f68d
* Result: SUCCEEDED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=82908ca1-6e41-4ab9-b2f0-121967d115c1%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: c8980a90026e78a3f5b47d186302d0f8dcc6f68d
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ef035f24-aa41-45a0-89fe-bf900e531495%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/model.py [hunk 1]
```diff
@@ -2152,7 +2148,7 @@ def update_approval_status(self, approval_status, approval_description=None):
         """Update the approval status for the model package
 
         Args:
-            approval_status (str or PipelineVariable): Model Approval Status, values can be
```
Comment (saumitravikram) [inline src/sagemaker/model.py:2155]: why removing PipelineVariable?

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: c8980a90026e78a3f5b47d186302d0f8dcc6f68d
* Result: SUCCEEDED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=69784fee-bb82-4680-9987-c3b472e5919d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request closed without merging**